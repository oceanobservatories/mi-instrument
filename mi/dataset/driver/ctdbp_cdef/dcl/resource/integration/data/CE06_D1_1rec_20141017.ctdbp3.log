2014/10/17 00:28:09.008 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/10/17 00:30:21.092 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/10/17 00:30:28.030 [ctdbp3:DLOGP6]: 15.0529,  3.60971,    0.998, 594, 161, 84, 17 Oct 2014 00:30:22
2014/10/17 00:31:04.042 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
