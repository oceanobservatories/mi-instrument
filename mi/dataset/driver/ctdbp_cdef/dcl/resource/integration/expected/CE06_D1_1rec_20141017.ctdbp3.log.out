
 subsite  | node  | sensor       | bin        | method      | time                 | deployment | internal_timestamp | port_timestamp       | preferred_timestamp | quality_flag | conductivity | date_time_string | dcl_controller_timestamp | pressure | temp
----------+-------+--------------+------------+-------------+----------------------+------------+--------------------+----------------------+---------------------+--------------+--------------+------------------+--------------------------+----------+---------
 CE06ISSM | SBD17 | 06-CTDBPC000 | 3622492800 | telemetered | 3622494628.029999733 |          1 |         3622494622 | 3622494628.029999733 |      port_timestamp |           ok |      3.60971 |             null |                     null |    0.998 | 15.0529

(1 rows)
